### PR TITLE
cleanup scripts

### PR DESCRIPTION
**Pull request by erasche**
This is more amenable to being run by administrators, since before the queries were simply copied from gxadmin.

It still probably needs to be documented that the user should set either [the pgpass file](https://www.postgresql.org/docs/9.3/libpq-pgpass.html) or the [environment variables](https://www.postgresql.org/docs/9.1/libpq-envars.html) but I leave it to you.

### CODE REVIEW

**Comment by anuprulez** [comment]
thanks a lot @erasche

File: extract_data.sh [hunk 1]
```diff
@@ -1,72 +1,23 @@
...
-	months=${1:-24}
-
-	read -r -d '' QUERY <<-EOF
+# Run the queries
+psql > data/tool-popularity.tsv <<-EOF
+	COPY (
 		SELECT
 			tool_id,
-			date_trunc('month', create_time AT TIME ZONE 'UTC')::date as month,
+			date_trunc('day', create_time AT TIME ZONE 'UTC')::date as day,
```
Comment (anuprulez) [inline extract_data.sh]: Since I am using the monthly usage data for tools, I will update it to `month` and not as `day`.

Comment (erasche) [reply extract_data.sh]: I thought you swapped to day since you'd requested that? Huh, ok :)

Comment (anuprulez) [reply extract_data.sh]: I tried with `day` but the learning time went up by a large margin. I am using non-linear regressor for that and the running time depends on the amount of data. Therefore, I switched back to `month`.

**Pull request merged**